bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/EpsonProjectorHandler.java

```diff
@@ -0,0 +1,451 @@
+                    break;
+                case KEY_CODE:
+                    remoteController.sendKeyCode(((DecimalType) command).intValue());
+                    break;
+                case LAMP_TIME:
+                    logger.error("'{}' is read only parameter", commmandType);
+                    break;
+                case LUMINANCE:
+                    remoteController.setLuminance(Luminance.valueOf(command.toString()));
+                    break;
+                case MUTE:
+                    remoteController.setMute((command == OnOffType.ON ? Switch.ON : Switch.OFF));
+                    break;
+                case OFFSET_BLUE:
+                    remoteController.setOffsetBlue(((DecimalType) command).intValue());
+                    break;
+                case OFFSET_GREEN:
+                    remoteController.setOffsetGreen(((DecimalType) command).intValue());
+                    break;
+                case OFFSET_RED:
+                    remoteController.setOffsetRed(((DecimalType) command).intValue());
+                    break;
+                case POWER:
+                    remoteController.setPower((command == OnOffType.ON ? Switch.ON : Switch.OFF));
+                    break;
+                case POWER_STATE:
+                    logger.error("'{}' is read only parameter", commmandType);
+                    break;
+                case SHARP:
+                    logger.warn("Set '{}' not implemented!", commmandType.toString());
+                    break;
+                case SOURCE:
+                    remoteController.setSource(Source.valueOf(command.toString()));
+                    break;
+                case SYNC:
+                    remoteController.setSync(((DecimalType) command).intValue());
+                    break;
+                case TINT:
+                    remoteController.setTint(((DecimalType) command).intValue());
+                    break;
+                case TRACKING:
+                    remoteController.setTracking(((DecimalType) command).intValue());
+                    break;
+                case VKEYSTONE:
+                    remoteController.setVerticalKeystone(((DecimalType) command).intValue());
+                    break;
+                case VPOSITION:
+                    remoteController.setVerticalPosition(((DecimalType) command).intValue());
+                    break;
+                case VREVERSE:
+                    remoteController.setVerticalReverse((command == OnOffType.ON ? Switch.ON : Switch.OFF));
+                    break;
+                default:
+                    logger.warn("Unknown '{}' command!", commmandType);
+                    break;
+            }
+
+        } catch (EpsonProjectorException e) {
+            logger.warn("Couldn't execute command '{}', {}", commmandType, e.getMessage());
+            // closeConnection();
```

Comment:
There is a TODO, what should be done and can it be completed before merging the PR?

Reply:
Same as above, I'd like to keep it like this as a reminder that this was in the 1.x binding